src/lammpsio/data.py

```diff
@@ -44,12 +45,19 @@ def __init__(self, filename, atom_style=None):
         self.filename = filename
         self.atom_style = atom_style
 
-    known_headers = ("atoms", "atom types", "xlo xhi", "ylo yhi", "zlo zhi", "xy xz yz")
-    unknown_headers = (
+    known_headers = (
+        "atoms",
+        "atom types",
+        "xlo xhi",
+        "ylo yhi",
+        "zlo zhi",
+        "xy xz yz",
         "bonds",
         "angles",
         "dihedrals",
         "impropers",
+    )
+    unknown_headers = (
```

Comment:
As mentioned at our meeting, you need to handle the bond types, angle types, etc. headers.